tasks/sep/026-fp-holocene-upgrade-fix/.env

```diff
@@ -0,0 +1,12 @@
+ETH_RPC_URL="https://ethereum-sepolia.publicnode.com"
+COUNCIL_SAFE=0xf64bc17485f0B4Ea5F06A96514182FC4cB561977
+FOUNDATION_SAFE=0xDEe57160aAfCF04c34C887B5962D0a69676d3C8B
+OWNER_SAFE=0x1Eb2fFc903729a0F03966B917003800b145F56E2
+SAFE_NONCE=""
+SIMULATE_WITHOUT_LEDGER=0 # 1
```

Comment:
This is causing issue with the justfile since the justfile is checking only if the `SIMULATE_WITHOUT_LEDGER` is set and not the value of the var.